public/config/configuration.json

```diff
@@ -2,7 +2,7 @@
   "BUSINESSES_URL": "https://dev.bcregistry.ca/business/auth/",
   "AUTH_URL": "https://dev.bcregistry.ca/business/auth/",
   "CREATE_URL": "https://business-create-dev.pathfinder.gov.bc.ca/businesses/create/",
-  "CORRECT_URL": "https://business-correct-dev.pathfinder.gov.bc.ca/businesses/correct/",
+  "EDIT_URL": "https://business-correct-dev.pathfinder.gov.bc.ca/businesses/edit/",
```

Comment:
Need to change host to `business-create...`.

Reply:
ah, right! Tho i think the routing might actually business-edit here. Just starting into OS stuff now, i've gotten the run down on what needs to be done 😅

Reply:
Fixed to -create in next commit. Can revisit this, if the base route can actually change under the same os project.